packages/block-editor/src/components/inspector-controls-tabs/position-controls-panel.js

```diff
@@ -17,38 +17,53 @@ import { default as InspectorControls } from '../inspector-controls';
 import { store as blockEditorStore } from '../../store';
 
 const PositionControlsPanel = () => {
-	const [ initialOpen, setInitialOpen ] = useState();
+	const { selectedClientIDs, hasPositionAttribute } = useSelect(
```

Comment:
```suggestion
	const { selectedClientIds, hasPositionAttribute } = useSelect(
```

The customary usage is "Ids".